app/src/main/java/com/uncookthebook/app/GoogleActivity.java

```diff
@@ -0,0 +1,11 @@
+package com.uncookthebook.app;
+
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount;
+import com.google.android.gms.auth.api.signin.GoogleSignInClient;
+
+public interface GoogleActivity {
+    GoogleSignInClient getGoogleClient();
+
+    GoogleSignInAccount getGoogleAccount();
+    void setGoogleAccount(GoogleSignInAccount account);
```

Comment:
maybe add some documentation to understand what the interface models and its methods do.